terraform/modules/paas/main.tf

```diff
@@ -59,3 +59,11 @@ resource cloudfoundry_route web_app_route {
   space    = data.cloudfoundry_space.space.id
   hostname = local.web_app_name
 }
+
+resource "cloudfoundry_user_provided_service" "logging" {
+  count            = var.logging
+  name             = var.logging_name
```

Comment:
`logging` is not very explicit. Could we replace this with something like `logging_service_name`?

Reply:
done...